src/ethTurin/IncreaseIterationPage.re

```diff
@@ -0,0 +1,14 @@
+open Globals;
+
+[@react.component]
+let make = () => {
+  let (increaseIteration, _increaseIterationTxState) =
+    AnimalActions.useIncreaseVoteIteration();
+  <Rimble.Box className=Styles.topBody>
```

Comment:
This was kept super simple on purpose. We need to think if we want to encourage our users to increase the iteration for us (currently the smart contract gives the person who increases the iteration 1% of the funds).